hack/lib/build/binaries.sh

```diff
@@ -219,6 +219,61 @@ os::build::internal::build_binaries() {
         local_ldflags+=" -s"
       fi
 
+      #Add Windows File Properties/Version Info and Icon Resource for oc.exe
+      if [[ "$platform" == "windows/amd64" ]]; then
+        os::build::version::get_vars
```

Comment:
Only remaining comment is it feels weird to be calling this explicitly but I have not given the call-chain thought to see why we don't already have the version here.